### DISCUSSION TITLE
Is there a wiki with the asset file format information?

### DISCUSSION THREAD

**Discussion by Lancelotbronner** [AssetRipper/AssetRipper#798] [General]
I'm trying to find the binary format of Unity's serialized files but I can't find anything about them.

Is there some sort of community-built wiki of reverse-engineered file formats? If not it would definitely be helpful and a simple GitHub pages static site could hold that information.

**Comment by ds5678** [comment]
No wiki. The format is proprietary and undocumented by Unity. Everything we know about it comes from Ghidra or trial and error. No one is likely to write anything into a web format. We generally view that as boring and tedious with little self benefit. The only thing we do is comment our source code because that has a direct benefit to us.

**Reply by Lancelotbronner** [reply]
Thanks, I guess as I try to figure this out I'll write them then!
